cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,24 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
+    cy.visit('https://conduit.mate.academy/user/login');
 
+    cy.get('h1')
+      .should('contain.text', 'Sign in');
+
+    // email entering
+    cy.get(':nth-child(1) > .form-control')
+      .type('bybeg@mailinator.com');
```

Comment:
Avoid using CSS selectors that rely on the structure of the HTML, as they can easily break when the structure of the page changes. Instead, use selectors that are less likely to change, such as `id`, `class`, or `name` attributes. For example, if the input has an id of `email`, you could use `cy.get('#email')`.